ugt_fwtools/utils.py

```diff
@@ -135,3 +136,70 @@ def username():
 
 def vivado_batch(source: str) -> None:
     subprocess.run(["vivado", "-mode", "batch", "-source", source, "-nojournal", "-nolog"]).check_returncode()
+
+
+def colored(text: str, color: Optional[str] = None, on_color: Optional[str] = None, attrs: Optional[str] = None) -> str:
```

Comment:
This function mimics [termcolor](https://pypi.org/project/termcolor/) that requires Python >= 3.8 (scripts still require to run on Python 3.6).